contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -458,29 +456,32 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
       // 2. blockHash at trigger block number was same as trigger time. This is an optional check which is
       // applied if DON sends non empty trigger.blockHash. Note: It only works for last 256 blocks on chain
       // when it is sent
-      emit ReorgedUpkeepReport(upkeepId, triggerID);
+      emit ReorgedUpkeepReport(upkeepId, transmitInfo.triggerID);
       return false;
     }
     return true;
   }
 
-  function _validateLogTrigger(uint256 upkeepId, bytes32 triggerID, bytes memory rawTrigger) internal returns (bool) {
+  function _validateLogTrigger(
+    uint256 upkeepId,
+    bytes memory rawTrigger,
+    UpkeepTransmitInfo memory transmitInfo
+  ) internal returns (bool, bytes32) {
     LogTrigger memory trigger = abi.decode(rawTrigger, (LogTrigger));
+    bytes32 dedupID = keccak256(abi.encodePacked(upkeepId, trigger.txHash, trigger.logIndex));
```

Comment:
I'm wondering if this could be the same as triggerID? Currently triggerID also includes hashing the checkBlockNum,checkBlockHash but not sure if it would make sense to unify this